src/ResourceManager/Network/Commands.Network/Models/PSUsage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Azure.Management.Network.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public partial class PSUsage
```

Comment:
Generator updated to create partial class for types. Allows to split custom and autogenerated code. [for example](https://github.com/EvgenyAgafonchikov/azure-powershell/pull/1/files#diff-20d5bc0e03f1ec7c99618d0a1d1e5ea0)